.dockerignore

```diff
@@ -6,9 +6,19 @@
 **/htmlcov
 **/__pycache__
 **/*.pyc
-**/.python-version
 **/.env
 **/.venv
 **/venv
 **/.coverage
 **/*.egg-info/
+
+# Python files
+/tests
+
+# Tooling effluent
+/htmlcov
```

Comment:
Isn't this already covered by the **/htmlcov on line 6? And I think ** matches any directories (including 0), so is it worth using that pattern for these folders too?